pupil_remote/Matlab_Python/PupilNetwork2.m

```diff
@@ -0,0 +1,25 @@
+% construct the pupil UDP
+hUDP = udp('127.0.0.1','LocalPort',8821,'Timeout',0.5);
+eyeProperties.timerPeriod = 0.05; % in seconds 0.05 = 50 ms
+
+fopen(hUDP);
+eyeProperties.isOpen = true;
+
+tic; fread(hUDP); elapsed=toc;
+if elapsed>=eyeProperties.timerPeriod
+    sprintf('Timeout for PupilNetwork UDP (%.2f sec): Check UDP connection ',elapsed);
+    fclose(hUDP);
+    eyeProperties.isOpen = false;
+else
+    eyeProperties.isConnected = true;
+end
+
```

Comment:
Running the `fread` test once does not work if Pupil Capture, and both matlab scripts (`pythonPupil.py`, `PupilNetwork2.m`) run on the same computer. A loop that reads multiple times to test the connection solves this problem.